### PR TITLE
Feat/containerize cg app

### PR DESCRIPTION
**Pull request by Mropat**
This PR adds/fixes ...

Containerfile and its configurations for running clinical-api

### How to test
- [x] Make sure container can be built

### Expected test outcome
- [x] Make sure container can be run with podman and not fail on startup

## Review
- [x] code approved by ES
- [x] tests executed by MR
- [x] "Merge and deploy" approved by MR
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [x] **PATCH** 

## Implementation Plan
- [ ] Merge this branch

### CODE REVIEW

**Comment by Mropat** [comment]
Preliminary ready.
Feedback appreciated

@eriksjolund I would appreciate if you had any comments regarding whether I need to declare user/uid in this file or if they should be set at runtime

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+FROM python:3.7-slim
```
Comment (eriksjolund) [inline Dockerfile:1]: I think it should be a fully qualified name


```
FROM python:3.7-slim
```
=>

```
FROM docker.io/library/python:3.7-slim
```
(If I remember correctly)

**Review by eriksjolund — APPROVED** [review]
Regarding the topic of how to deal with secrets inside a Dockerfile, I don't have any good answer right now. I need to learn more. Maria we discussed this briefly before.

File: Dockerfile (hunk 1 above)
Comment (Mropat) [inline Dockerfile:1]:
```suggestion
FROM docker.io/library/python:3.7-slim
```

File: Containerfile [hunk 2]
```diff
@@ -0,0 +1,40 @@
+FROM python:3.7-slim
```
[the remark this answers is not in the document]
Comment (Mropat) [reply Containerfile]: Done

**Comment by Mropat** [comment]
Will deploy this as is, and modify if needed

**Pull request merged**